habitat_baselines/common/policy_registry.py

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env python3
+
+# Copyright (c) Facebook, Inc. and its affiliates.
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+
+r"""PolicyRegistry is extended from habitat.Registry to provide
+registration for policy, while keeping Registry in habitat core intact.
+
+Import the policy registry object using
+
+``from habitat_baselines.common.policy_registry import policy_registry``
+
+Decorator for registry different kind of classes with unique keys
+
+- Register a policy: ``@policy_registry.register_policy``
+"""
+
+from typing import Optional
+
+from habitat.core.registry import Registry
+
+
+class PolicyRegistry(Registry):
+    @classmethod
+    def register_policy(cls, to_register=None, *, name: Optional[str] = None):
+        r"""Register a RL policy to registry with key 'name'.
+
+        Args:
+            name: Key with which the policy will be registered.
+                If None will use the name of the class.
+
+        """
+        from habitat_baselines.rl.ppo.policy import Policy
+
+        return cls._register_impl(
+            "policy", to_register, name, assert_type=Policy
+        )
+
+    @classmethod
+    def get_policy(cls, name):
```

Comment:
Also should have a doc string.

Reply:
Updated
**Note:** other `get_somethint` methods in the Registry and BaselineRegistry don't have docstrings.

Reply:
Can't hurt 👍